README.pod

```diff
@@ -268,6 +269,15 @@ in a file named `Build.pm` at the root of your distribution:
         }
     }
 
+NOTE: You may use installed modules inside the build hook, but those modules
```

Comment:
I'm not sure this requires stating. For instance this same concept is *implied* for test files -- we don't need to tell the user they need to declare test depends in `test-depends` or how to use `use lib ...`

Reply:
Maybe, but how many people think or even about the build step?

Reply:
I've submitted another commit.

Reply:
Most people don't *need* to think about the build step. Additionally we had hoped to get rid of `Build.pm6`, or at least change how it works. Having a file in the root directory of a perl6 distribution causes issues because that same directory contains a META6.json (and that `Build.pm6` isn't listed anywhere inside there, so we have to implement hacky ways to load/see it). This could obviously be improved by making people list the `Build.pm6` in the META6.json somewhere, but then we might as well also allow it to be named anything and be in a directory. HOWEVER -- as noted [here](https://deathbyperl6.com/perl-toolchain-summit-2018-meta6-dependency-hello/) there is supposed for a `builder` key in META6.json that can mostly fulfill this purpose (still would need the mapping of `"Build" : "./Build.pm"` somewhere in META6.json though).